orchestrator/src/handlers/price/sources/pyth.rs

```diff
@@ -58,25 +52,25 @@ impl PricingDataSource for Pyth {
         ))
     }
 
-    /// get latest price data for a currency from pyth api
-    async fn get_price(ticker: String) -> Result<f64> {
+    async fn get_proof(ticker: String) -> Result<ProofTypes> {
```

Comment:
Why remove comments?
Please leave comments on functions to provide further context.

Here, it's quite straight forward, but just general advice.